africhi.csl

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>africhi</title>
+    <id>http://www.zotero.org/styles/africhi</id>
+    <link href="http://www.zotero.org/styles/africhi" rel="self"/>
+    <link href="http://www.acm.org/sigs/publications/proceedings-templates" rel="documentation"/>
+    <author>
+      <name>Naeem Esfahani</name>
+      <email>nesfaha2@gmu.edu</email>
+      <uri>http://mason.gmu.edu/~nesfaha2/</uri>
+    </author>
```

Comment:
Did you see the description at the link above?

I'm not a 100% sure which ACM SIG CSL style you based africhi.csl on. But, for example, if you based this style off of https://github.com/citation-style-language/styles/blob/master/acm-sig-proceedings.csl, the "template" link would be 

```xml
<link href="http://www.zotero.org/styles/acm-sig-proceedings" rel="template"/>
```